packages/test/service-load-test/src/nodeStressTest.ts

```diff
@@ -150,22 +169,56 @@ async function runnerProcess(
     testId: string,
 ): Promise<number> {
     try {
+        const beganMs = Date.now();
         const runConfig: IRunConfig = {
             runId,
             testConfig: profile,
+            leaveDelayMs: randomIntBetween(15000, 25000),
         };
+        // TODO: consider moving "magic" numbers to JSON file
```

Comment:
Do you expect to resolve all TODO comments in this PR? I prefer not to check these in.

Reply:
And in this case - yes I agree please put these numbers in the config file :)